qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -120,6 +120,9 @@ def check_network_config(session_serial):
             win_driver_installer_test.device_name_list,
             win_driver_installer_test.device_hwid_list,
         ):
+            if not params.get("boot_with_viomem"):
```

Comment:
How about using `if params.get("boot_with_viomem", "no") == "yes":` like above.

Reply:
Sure， just a quick question the reason why we propose this is to unity the uasge like above？

Reply:
if not params.get("boot_with_viomem"): means the variable is not set, if somewhere the variable is set to some value, it would cause the wrong results.

Reply:
> if not params.get("boot_with_viomem"): means the variable is not set, if somewhere the variable is set to some value, it would cause the wrong results.

Make sense， thanks for the explanation!